extensions/pkg/controller/worker/controller.go

```diff
@@ -117,7 +117,7 @@ func wantMachineWatch(isAutonomousShootCluster bool, restMapper meta.RESTMapper)
 }
 
 func machineAPIPresent(restMapper meta.RESTMapper) (bool, error) {
-	if _, err := restMapper.KindsFor(schema.GroupVersionResource{Group: machinev1alpha1.GroupName, Version: machinev1alpha1.SchemeGroupVersion.Version, Resource: "machines"}); err != nil {
+	if _, err := restMapper.KindsFor(machinev1alpha1.SchemeGroupVersion.WithResource("machines")); err != nil {
```

Comment:
The `k8s.io/apimachinery/pkg/runtime/schema` import is now unused

Reply:
:(